itables/html/datatables_template_connected.html

```diff
@@ -1,12 +1,11 @@
 <div class="itables">
 <style></style>
 <table id="table_id"><thead><tr><th>A</th></tr></thead></table>
-<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.13.1/css/jquery.dataTables.min.css">
+<link href="https://cdn.datatables.net/v/dt/jszip-3.10.1/dt-2.0.1/b-3.0.0/b-html5-3.0.0/datatables.min.css" rel="stylesheet">
 <script type="module">
     // Import jquery and DataTable
-    import 'https://code.jquery.com/jquery-3.6.0.min.js';
-    import dt from 'https://cdn.datatables.net/1.12.1/js/jquery.dataTables.mjs';
-    dt($);
+    import 'https://code.jquery.com/jquery-3.7.1.min.js';
+    import 'https://cdn.datatables.net/v/dt/jszip-3.10.1/dt-2.0.1/b-3.0.0/b-html5-3.0.0/datatables.min.js';
```

Comment:
@AllanJard, this works in a web browser and in Jupyter, but unfortunately not in VS Code (which complains that `DataTable` below is not defined). Would you have `.mjs` extensions for the bundles ? Or do you think you could export `DataTable` explicitly in the `.js` file? Thanks!

Reply:
I am moving this conversation to #237